specification/cognitiveservices/Language.AnalyzeText/common.tsp

```diff
@@ -0,0 +1,80 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace Language.AnalyzeText;
+
+#suppress "@azure-tools/typespec-azure-core/long-running-polling-operation-required" "template"
+@Foundations.Private.needsRoute
+@post
+@doc("Long running operation template to match Azure Language.")
+op LanguageLongRunningOperation<
+  TParams extends TypeSpec.Reflection.Model,
+  TStatusResult extends TypeSpec.Reflection.Model,
+  TStatusError = Foundations.Error,
+  Traits extends TypeSpec.Reflection.Model = {},
+  TErrorResponse = ErrorResponse
+> is Foundations.Operation<
+  TParams &
+    Azure.Core.Traits.Private.TraitProperties<
```

Comment:
Let's not use private components of the azure core lib

Reply:
We tried using off the shelf definition for our operations but nothers seemed to fit how the service was described in openapi, so we followed what openai has for their operations and modified to fit our needs